.github/workflows/linux-eic-shell.yml

```diff
@@ -184,39 +184,12 @@ jobs:
       - run: |
           diff ${{ matrix.detector_config }}_right_flags.json ${{ matrix.detector_config }}_flags.json
```

Comment:
Isn't this diff line what fails?

Reply:
IDK whose line is that =)

Json file has both default valus and user values. So single file from reco-flags has all that is needed to diff. Then there is a GUI now.

Reply:
Sorry, from phone, cant run blame and investigate

Reply:
fixed